backend/src/TrayMenu.ts

```diff
@@ -85,15 +90,19 @@ export default class TrayMenu {
         const location = hostName(connection, this.privateIP)
         result.push({
           label: connection.name,
-          icon: connection.active ? iconConnected : iconOnline,
+          icon: connection.active ? iconConnected : connection.online ? iconOnline : iconOffline,
```

Comment:
Ternary in a ternary.  Is that a pattern we want to keep?

Reply:
I'm okay with it.  :)